CHANGES.md

```diff
@@ -19,6 +19,7 @@ Language improvements:
   - New grammar for attributes
   - Added support for quoted identifiers, implicit parameters, and property wrapper projections
   - Support for more complex expressions in string interpolation
+- enh(swift) New grammar for types (#2920) [Steven Van Impe][]
```

Comment:
```suggestion
- enh(swift) Better highlight type identifiers (#2920) [Steven Van Impe][]
```

Reply:
Or something, I have no idea at all what "new grammar for types" means.. it's not a new grammar, it's just some new rules, so we should try and describe the actual change that users will see.  So I looked at the notes from your PR to whip up that suggestion.

Reply:
Done